src/components/AuthButton.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../views/Button/index";
+import { Logout } from "../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onLogout = this.onLogout.bind(this);
+  }
+
+  onLogout(event) {
+    const { dispatch } = this.props;
+    event.preventDefault();
+    dispatch(Logout());
+  }
+
+  render() {
+    const { isAuthenticated } = this.props;
+
+    return (
+      <div>
+        {isAuthenticated ? (
+          <div className="navbar-text">
+            <ul className="navbar-nav mr-auto">
+              <li className="nav-item dropdown">
+                <a
+                  className="nav-link dropdown-toggle"
+                  href="/"
+                  id="navbarDropdownMenuLink"
+                  data-toggle="dropdown"
+                  aria-haspopup="true"
+                  aria-expanded="false"
+                >
+                  <i className="fas fa-user-circle" />
+                </a>
+                <div
+                  className="dropdown-menu"
+                  aria-labelledby="navbarDropdownMenuLink"
+                >
+                  <a className="dropdown-item" href="profile.html">
+                    My profile
+                  </a>
+                  <a href="/" className="dropdown-item" onClick={this.onLogout}>
+                    Logout
```

Comment:
Where exactly on the page do we have `profile` and `logout`? I cant seem to find the two on the UI

Reply:
@kimotho-njoki: Once a user has logged in, you will be able to see an avatar with a drop down that will have a profile link and log out link. Currently, the profile link is not doing anything since the front end feature is not implemented.
<img width="1019" alt="screen shot 2018-10-15 at 21 28 31" src="https://user-images.githubusercontent.com/30254491/46970307-65346c00-d0c1-11e8-9a50-b5894a566b67.png">